opm/material/common/UniformXTabulated2DFunction.hpp

```diff
@@ -388,6 +423,8 @@ class UniformXTabulated2DFunction
 
     // the position of each vertical line on the x-axis
     std::vector<Scalar> xPos_;
+    std::vector<Scalar> yPos_;
+    bool doLeft_;
```

Comment:
probably this should be called `extrapolateLeft_`? shouldn't there also be a analog for the right side. this may even have some practical relevance, since some tables reorder the values from descending to ascending order.